openapi-core/src/main/java/io/ballerina/openapi/core/generators/common/OASModifier.java

```diff
@@ -114,9 +115,111 @@ public OpenAPI modifyWithBallerinaConventions(OpenAPI openapi, Map<String, Strin
         openapi = modifyOASWithSchemaName(openapi, nameMap);
         modifyOASWithParameterName(openapi);
         modifyOASWithObjectPropertyName(openapi);
+        modifyOASWithObjectPropertyNameInlineObjectSchema(openapi);
```

Comment:
Can't we combine this to the `modifyOASWithParameterName` function? That function already loop through the path parameters. So if we do this modification within the same visitor then it will efficient. Also I don't see a separate use case for this function, if we want to sanitize the parameters then we need to call both `modifyOASWithParameterName` and `modifyOASWithObjectPropertyNameInlineObjectSchema`

May be we can rename this function properly since the `modifyOASWithObjectPropertyNameInlineObjectSchema` sanitize the inline response schemas as well

Reply:
While implementing this had a same thought of combining handling inline object schema in parameter with this   `modifyOASWithParameterName`  function. avoiding combing this by assuming inline object schema is not in a common case, will check whether we can combing these two together.

Reply:
I checked the ability of combining, there are concerned with combing with the implementation `modifyOASWithParameterName`—this function only modifies parameter names.  `modifyOASWithObjectPropertyNameInlineObjectSchema` only modifies properties of object types.  If we incorporate these two, there can be additional unnecessary travers like, if there are no object types in the inline schema, the person would still need to go through the logic since it's tied to the parameter name modifier. 
Therefore , I thought to having both function separately and will modify the meaning full name, WDYT?

Reply:
Updated with https://github.com/ballerina-platform/openapi-tools/pull/1775/commits/b30ae39666d29f909a32bb60da3b06d4eb1ba1a3